src/app/activity/activity.ts

```diff
@@ -1,9 +1,19 @@
+import {firestore} from 'firebase/app';
+import Timestamp = firestore.Timestamp;
+
 export class Activity {
-  user: string;
-  user_nickname: string;
-  date: Date;
-  individual: string;
+  action?: string;
+  activity_date: Timestamp;
+  followers: [string];
   individual_id: string;
-  text: string;
-  name: string;
+  individual_name: string;
+  observation_id?: string;
+  phenophase: string;
+  phenophase_name: string;
+  source: string;
```

Comment:
`source` could have a similar restriction like `type`

Reply:
yes thanks, probably would make sense to set an type alias for these ones like 'source' used all over the project.
Maybe it would make sense to centralise the datamodell in a package. Would need to read up what if there is a best practice there.